docs/reference.rst

```diff
@@ -0,0 +1,113 @@
+.. _sec_ref:
+
+Reference
+=========
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _sec_ref_config_file:
+
+Configuration File
+------------------
+
+Example configuration:
+
+.. code-block:: cfg
+
+  [client]
+  hawkbit_server            = 127.0.0.1:8080
+  ssl                       = false
+  ssl_verify                = false
+  tenant_id                 = DEFAULT
+  target_name               = test-target
+  auth_token                = bhVahL1Il1shie2aj2poojeChee6ahShu
+  #gateway_token            = bhVahL1Il1shie2aj2poojeChee6ahShu
+  bundle_download_location  = /tmp/bundle.raucb
+  retry_wait                = 60
+  connect_timeout           = 20
+  timeout                   = 60
+  log_level                 = debug
+
+  [device]
+  product                   = Terminator
+  model                     = T-1000
+  serialnumber              = 8922673153
+  hw_revision               = 2
+  key1                      = value
+  key2                      = value
```

Comment:
Do we need this here? This is the same as config.conf.example in the project's root.

Reply:
I'd tend to keep it as people reading the documentation online might not have the source present. I'll shorten this a little